ci/deploy.yml

```diff
@@ -226,3 +226,23 @@ components storybook deploy dev manual:
 #       - /builds/satoshilabs/trezor/trezor-suite/packages/integration-tests/projects/components-storybook/snapshots
 #       - /builds/satoshilabs/trezor/trezor-suite/packages/integration-tests/projects/components-storybook/screenshots
 #   needs: []
+
+# Suite desktop autoupdate deploy
+suite-desktop deploy autoupdate test:
+  stage: deploy to dev
+  needs:
+    - suite-desktop build mac codesign
+    - suite-desktop build linux codesign
+    - suite-desktop build windows codesign
+  only:
+    refs:
+      - codesign
+  when: manual
+  variables:
+    DEPLOY_DIRECTORY: ${DEPLOY_BASE_DIR}/suite-desktop/${CI_BUILD_REF_NAME}
```

Comment:
Should it go to suite-desktop/codesign?

Reply:
This job was already there. I just moved it to deploy.yml from release.yml. `${CI_BUILD_REF_NAME}` is `codesign` in this case, and it was deployed there a couple of times.